package.json

```diff
@@ -6,9 +6,11 @@
   "scripts": {
     "css:test": "rimraf ./backstop_data/bitmaps_test && backstop test --config=config/backstop/backstopConfig",
     "css:reference": "rimraf ./backstop_data/bitmaps_reference && backstop reference --config=config/backstop/backstopConfig",
-    "deploy": "bash ./config/hooks/deploy.sh",
+    "deploy": "git add . && git commit -m 'solution' && git push origin develop && bash ./config/hooks/deploy.sh",
```

Comment:
deploy.sh is already committing /src dir. Do we need to commit anything else before pushing to github?